src/ext/rapidjson/document.h

```diff
@@ -36,8 +36,10 @@ RAPIDJSON_DIAG_OFF(4127) // conditional expression is constant
 #elif defined(__GNUC__)
 RAPIDJSON_DIAG_PUSH
 RAPIDJSON_DIAG_OFF(effc++)
+#if __GNUC__ >=6
+  RAPIDJSON_DIAG_OFF(terminate)

```

Comment:
Just curious, related to some difficulties with TRT ?

Reply:
do not remember, seems ok w/o, removed